tools/build/_/index.ts

```diff
@@ -101,7 +102,14 @@ export function resolve(
   return from(glob(pattern, options))
     .pipe(
       catchError(() => EMPTY),
-      switchMap(files => from(files)),
+      concatAll(),
+
+      /* Build overrides */
+      !process.argv.includes("--all")
```

Comment:
We don't have any overrides so we should just remove the whole portion starting with `/* Build overrides */` and ending with `: identity,`.

Reply:
I also noticed that there is a new `build:all` run script in the package.json that appends the `--all` arg. Do you want to me remove that as well?

Reply:
Yeah we should remove that since it won't do anything.